Description: fix man section of the generated manpages
 On Debian, manual pages about Perl modules need to nbe with '3pm' extension as
 per Debian Perl policy.
Forwarded: not-needed
Author: Gunnar Wolf <gwolf@debian.org>
Last-Update: 2007-08-18

--- a/lib/ModPerl/BuildMM.pm
+++ b/lib/ModPerl/BuildMM.pm
@@ -281,7 +281,7 @@ sub ModPerl::BuildMM::MY::postamble {
                 $man =~ s!/!::!g;
 
                 push @target,
-                    '$(NOECHO) $(POD2MAN_EXE) --section=3 ' .
+                    '$(NOECHO) $(POD2MAN_EXE) --section=3pm ' .
                     "$podpath \$(INST_MAN3DIR)/$man.\$(MAN3EXT)"
             }
         }
